ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -179,6 +183,7 @@ class CellEditorOverlay extends Component {
               onSelectGraphType={this.handleSelectGraphType}
               onCancel={onCancel}
               onSave={this.handleSaveCell}
+              isSavable={queriesWorkingDraft.every(isQuerySavable)}
```

Comment:
If `<OverlayControls>` is used in other components, the logic should be flipped here, I think. That way if if the bool is `undefined` in `<OverlayControls>` elsewhere, it will pass through as falsy (`undefined`) to `<ConfirmButtons>.isDisabled`.

And then here, the prop could be called something like `isUnsavable` that would map directly onto the necessary value of `isDisabled` in `<ConfirmButtons>`. (And in light of of that, it may be useful upon second thought to consider a semantics of `isQueryFullyQualified` for the function name.) So, all told, this line might read `isUnsavable={!queriesWorkingDraft.every(isQueryFullyQualified)}` (or its inverse with a different method than `.every` or flipping the logic in the function itself and calling it something like `isQueryNotFullyQualified`.